python/composio/tools/toolset.py

```diff
@@ -628,6 +650,22 @@ def _process_schema_properties(self, action: Action, properties: t.Dict) -> t.Di
             type_="schema",
         )
 
+    def _merge_processors(self, processors: ProcessorsType) -> None:
```

Comment:
Consider adding a docstring for the new `_merge_processors` method to explain its purpose and usage. This will help other developers understand the method's functionality and how to use it correctly.